application-templates/starter/package.json

```diff
@@ -33,7 +33,7 @@
   "devDependencies": {
     "@commercetools-frontend/jest-preset-mc-app": "14.3.2",
     "@commercetools-frontend/mc-scripts": "14.3.0",
-    "@testing-library/react": "8.0.7",
+    "@testing-library/react": "9.1.0",
```

Comment:
Ah, interesting:

```
It looks like you're using a version of react-dom that supports the "act" function, but not an awaitable version of "act" which you will need. Please upgrade to at least react-dom@16.9.0 to remove this warning.
```

Reply:
I guess the question is: do we want to bump React as well or should we wait a bit?

Reply:
I would argue we wait and do it in the breaking release of v15 then.